docs/transpile/defaults-and-configuration-options.ipynb

```diff
@@ -0,0 +1,270 @@
+      "text/plain": [
+       "<Figure size 2299.9x1204 with 1 Axes>"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager\n",
+    "\n",
+    "# Specify the system to target\n",
+    "backend = FakeSherbrooke()\n",
+    "\n",
+    "# Transpile the circuit\n",
+    "pass_manager = generate_preset_pass_manager(optimization_level=1, backend=backend)\n",
+    "transpiled_circ = pass_manager.run(qc)\n",
+    "\n",
+    "# View the transpiled circuit\n",
+    "transpiled_circ.draw(output='mpl', idle_wires=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6c0e9d2d-511e-4948-9f48-3b7b0fe56d7a",
+   "metadata": {},
+   "source": [
+    "## All available parameters\n",
+    "\n",
+    "Following are all of the available parameters for the [`generate_preset_pass_manager`](/api/qiskit/transpiler_preset#qiskit.transpiler.preset_passmanagers.generate_preset_pass_manager) function.  There are two classes of arguments: those that describe the target of compilation, and those that influence how the transpiler works.\n",
+    "\n",
+    "\n",
+    "All parameters except `optimization_level` are optional.  For full details, see the [Transpiler API documentation](/api/qiskit/transpiler#transpiler-api).\n",
+    "\n",
+    "- `optimization_level` (int) - How much optimization to perform on the circuits. Integer in the range (0 - 3). Higher levels generate more optimized circuits, at the expense of longer transpilation time. See the [Set optimization topic](set-optimization) for more details.\n",
+    "\n",
+    "### Parameters used to describe the compilation target:\n",
+    "\n",
+    "These arguments describe the system that the input circuit will be run on.  For example, they may be used to ensure that the returned circuit can be run on the specified system, where the transpiler maps the circuit's virtual qubits to the physical qubits which may have limited connectivity as specified in the `coupling_map`, `basis_gates` that specify which single- and two-qubit gates are calibrated for the system, and possibly their error rates.\n",
```

Comment:
```suggestion
    "### Parameters used to describe the compilation target\n",
    "\n",
    "These arguments describe the system that the input circuit will be run on.  For example, they can be used to ensure that the returned circuit can run on the specified system, where the transpiler maps the circuit's virtual qubits to the physical qubits which may have limited connectivity as specified in the `coupling_map`, `basis_gates` that specify which single- and two-qubit gates are calibrated for the system, and possibly their error rates.\n",
```

Reply:
Made a couple grammar tweaks but am having trouble parsing the sentence in line 114 (starting with For example) -- possible to rewrite/split up?

Reply:
I rewrote the sentence
> These arguments describe the target system for circuit execution, including information such as the coupling map of the system (which describes the connectivity of the qubits), the basis gates supported by the system, and even the error rates of the gates.